src/Mappers/BankTransactionMapper.php

```diff
@@ -144,11 +146,14 @@ private static function setBankTransactionLineBaseFields(
         Base $line
     ): void {
         $line->setId($lineElement->getAttribute("id"));
-        $line->setDescription(self::getField($lineElement, "description"));
         $value = self::getField($lineElement, 'value');
         $line->setValue(Util::parseMoney($value, $bankTransaction->getCurrency()));
         $line->setInvoiceNumber(self::getField($lineElement, "invoicenumber"));
 
+        $description = self::getField($lineElement, "description");
+        if ($description) {
```

Comment:
This is always a bit tricky because `"0"` would be a valid description. I recommend explicit `null` checks instead.

Reply:
Do you mean an explicit `null` check for description only, or in general?